pkg/session/bootstrap.go

```diff
@@ -769,7 +769,18 @@ const (
         updated_at datetime,
         status varchar(128),
         description text,
-        primary key(module, name))`
+        primary key(module, name));`
+
+	// CreateWorkloadValuesTable is a table to store workload-based learning values for tidb.
+	CreateWorkloadValuesTable = `CREATE TABLE IF NOT EXISTS mysql.workload_values (
```

Comment:
Is there a standard naming convention that we use for these new tables that are TiDB extensions? I see many other cases of these, non-MySQL, tables where we prefix the table name with `tidb_`, `pd_`, `tikv_` or similar across the `[mysql](https://docs.pingcap.com/tidb/stable/mysql-schema#ttl-related-system-tables)` and `[performance_schema](https://docs.pingcap.com/tidb/stable/performance-schema#tables-that-are-tidb-extensions)` schemas.

I am concerned that this kind of unprefixed table name could lead to future conflicts and compatibility issues if MySQL introduces this table name in the future.

Reply:
Has been changed to tidb_workload_values